config/config.go

```diff
@@ -138,10 +138,10 @@ type GrafanaConfig struct {
 
 // JaegerConfig describes configuration used for jaeger links
 type JaegerConfig struct {
-	URL              string `yaml:"url"`
-	Service          string `yaml:"service"`
-	JaegerService    string `yaml:"jaeger_service"`
-	ServiceNamespace string `yaml:"service_namespace"`
+	URL           string `yaml:"url"`
+	Service       string `yaml:"service"`
+	JaegerService string `yaml:"jaeger_service"`
+	Namespace     string `yaml:"namespace"`
```

Comment:
did you change these settings in the operator too? 

https://github.com/kiali/kiali/pull/1022/files#diff-f8f6c9abc6c5969045b61451c949e922R174
https://github.com/kiali/kiali/pull/1022/files#diff-f8f6c9abc6c5969045b61451c949e922R181
https://github.com/kiali/kiali/pull/1022/files#diff-c924dfb634d9c40dbd97889668ab1efdR59

Reply:
No @jmazzitelli, because I was checking why the service Namespace wasn't work , with this change is working, I'll update the code asap